packages/openapi-fetch/src/index.js

```diff
@@ -72,7 +72,7 @@ export default function createClient(clientOptions) {
             });
     }
 
-    const serializedBody = body === undefined ? undefined : bodySerializer(body);
+    const serializedBody = body === undefined ? undefined : bodySerializer(body, headers);
```

Comment:
So these are the headers unique to this request (probably could be named better). Below on line `92`, we merge all the headers together. We should probably move that up, adn pass that in here to `bodySerializer`. That way it takes into account headers that users have set in their `createClient()` call, and it handles the normalization for you so it’s always a `Header` type (and you aren’t dealing with what the user passed in)